### PR TITLE
Fix Windows ARM release build break

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

**Comment by jkotas** [comment]
Introduce by https://github.com/dotnet/coreclr/pull/15903
cc @CarolEidt

File: src/jit/morph.cpp [hunk 1]
```diff
@@ -4967,7 +4967,7 @@ GenTreePtr Compiler::fgMorphMultiregStructArg(GenTreePtr arg, fgArgTabEntryPtr f
         if (lcl != nullptr)
         {
             // Its fields will need to accessed by address.
-            lvaSetVarDoNotEnregister(lcl->gtLclNum, DNER_IsStructArg);
+            lvaSetVarDoNotEnregister(lcl->gtLclNum DEBUG_ARG(DNER_IsStructArg));
```
Comment (ahsonkhan) [inline src/jit/morph.cpp]: Should there be a comma?
i.e. `lvaSetVarDoNotEnregister(lcl->gtLclNum, DEBUG_ARG(DNER_IsStructArg));`

Comment (jkotas) [reply src/jit/morph.cpp]: No. Take a look at other `lvaSetVarDoNotEnregister` calls in this file.

**Comment by CarolEidt** [comment]
@jkotas thanks for taking care of this

**Pull request merged**